src/parser/bison_parser.y

```diff
@@ -1176,19 +1188,22 @@ interval_literal : INTVAL duration_field { $$ = Expr::makeIntervalLiteral($1, $2
   free($2);
   $$ = Expr::makeIntervalLiteral(duration, $3);
 }
+| INTERVAL INTVAL datetime_field {
+  $$ = Expr::makeIntervalLiteral($2, $3);
+}
```

Comment:
Do you have a use case for this version? According to the SQL:2023 standard, only 
```
INTERVAL 'INTVAL' duration_field
```
would be valid (note the quotation marks). However, we supported some more versions according to Postgres. `INTERVAL INTVAL datetime_field` does not seem to work in Postgres (see [DB Fiddle](https://www.db-fiddle.com/f/bo7ZGJ9McxSjUESbWUfn58/0), statements in comments raise errors).